pocs/state/states/default/tracking.py

```diff
@@ -1,14 +1,31 @@
+from pocs.utils import current_time
+
+
 def on_enter(event_data):
     """ The unit is tracking the target. Proceed to observations. """
     pocs = event_data.model
-    pocs.say("Checking our tracking")
+    pocs.next_state = 'parking'
+
+    # If we came from pointing then don't try to adjust
+    if event_data.transition.source != 'pointing':
```

Comment:
I'm confused about pointing vs tracking vs observing w.r.t. moving / adjusting the mount.

Reply:
In what sense? For what the mount reports or for the actual state? Maybe some documentation of the states will eventually help with this.